release/katelem/katelem.ann-latn.getat/source/katelem.ann-latn.getat.model.kps

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package>
+  <System>
+    <KeymanDeveloperVersion>13.0.115.0</KeymanDeveloperVersion>
+    <FileVersion>12.0</FileVersion>
+  </System>
+  <Options>
+    <ExecuteProgram></ExecuteProgram>
+    <ReadMeFile>readme.htm</ReadMeFile>
+    <MSIFileName></MSIFileName>
+    <MSIOptions></MSIOptions>
+  </Options>
+  <StartMenu>
+    <Folder></Folder>
+    <Items/>
+  </StartMenu>
+  <Info>
+    <Name URL="">Getat</Name>
+    <Copyright URL="">© 2020 Rogers Katelem Edeh</Copyright>
+    <Author URL="mailto:katelem247@gmail.com">Rogers Katelem Edeh</Author>
+    <Version URL="">1.0</Version>
+  </Info>
+  <Files>
+    <File>
+      <Name>..\build\katelem.ann-latn.getat.model.js</Name>
+      <Description></Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.js</FileType>
+    </File>
+    <File>
+      <Name>welcome.htm</Name>
+      <Description></Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.htm</FileType>
+    </File>
+    <File>
+      <Name>readme.htm</Name>
+      <Description></Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.htm</FileType>
+    </File>
+  </Files>
+  <Keyboards/>
+  <LexicalModels>
+    <LexicalModel>
+      <Name>Getat</Name>
+      <ID>katelem.ann-latn.getat</ID>
+      <Languages>
+        <Language ID="ann-Latn">Obolo (Andoni)</Language>

```

Comment:
Just checking if you were wanting `Andoni` as the region. The Keyman Developer convention is to list the *script, region* names in parens, so `ann-Latn` would be `Obolo (Latin)`.

Reply:
I checked available regions, and I think the full tag would be
`ann-Latn-NG` for "Obolo (Latin, Nigeria)"

But more importantly, you'll want to make sure the tag you use matches the one for the keyboard.

Reply:
"Andoni" is an alternate name for Obolo, but it can be removed if it creates conflict in the system.

Keyman Developer 14 beta said that ann-Latn-NG is not canonical, during compilation, so I changed it to ann-Latn as it suggested. You can help change to what works best.

As for the words that are repeated in the wordlist, they can be merged or the one with lower frequency can be deleted.

Thanks.